### DISCUSSION TITLE
pull to refresh scroll bug

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3627] [Mobile] [answered]
to reproduce

1. scroll to right
2. pull to refresh

recorded in this pr https://github.com/flutter/flutter/pull/78919/files

https://user-images.githubusercontent.com/39104740/112762282-81b18e00-9007-11eb-9ba5-ef519d0e6a74.mp4

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @nt4f04uNd!

Could you please enable line wrapping in Settings->Code Options and double check if the bug is reproducible then? Could you also share the device model and Android OS version you are using?

**Reply by nt4f04uNd** [reply]
it expectedly didn't reproduce, since there's no way to scroll horizontally with wrapped lines

xiaomi redmi note 5 android 9
